boefjes/packaging/scripts/build-debian-package.sh

```diff
@@ -14,4 +14,4 @@ EOF
 
 dpkg-buildpackage -us -uc -b
 
-mv /${PKG_NAME}_${RELEASE_VERSION}_*.deb /app/build/
+mv /"${PKG_NAME}"_"${RELEASE_VERSION}"_*.deb /app/build/
```

Comment:
Shouldn't this be `"/${PKG_NAME}_${RELEASE_VERSION}_*.deb"` or is that essentially the same thing?

Reply:
Shellcheck tells you to add the quotes to prevent globbing if variable contain a glob character.  So if $PKG_NAME would be "pkg*name" would expand the * if it matches a filename. This wouldn't happen in this case, but shellcheck can't know that.

If we would add quotes around the *.deb it would not expand the version number anymore, but that's what we want.